packages/headless/src/features/result-preview/result-preview-state.ts

```diff
@@ -0,0 +1,11 @@
+export interface ResultPreviewState {
+  uniqueId: string;
+  content: string;
+}
```

Comment:
Only a single result preview is in state at a time.

Reply:
Could a preview be so large that it could cause problems with the redux devtools? In that case we could 'omit' the preview part of the state (in store.ts)

Reply:
Yeah I really think we should <omit> them for tooling, at least.